### ISSUE TITLE
Add endpoint to "track" triumphs like in-game

### ISSUE THREAD

**Issue by lowlines** [Bungie-net/api#668]
It would be great if there was a way to "track" a record/triumph like you can in-game via the APIs.

**Comment by ckhicks** [comment]
Would you want to get updates as it changes? I'm intrigued.

**Comment by vthornheart-bng** [comment]
As in changing the currently tracked triumph according to the game?  Or as in getting some kind of live update capability?

**Comment by ArkahnX** [comment]
I think @lowlines means something like the ability to mark bounties as tracked such as you could in D1 via the API

and I think they are wanting this to apply to items in the pursuits inventory that can be tracked, as well as Triumphs (which can be tracked)

**Comment by soren42** [comment]
Also to know when a location is tagged on the map in-game.  (That would be particularly useful to @lowlines.)

**Comment by vthornheart-bng** [comment]
Ah, good info!  I'll need to investigate those cases.  Sounds like there's three, to summarize?

* Ability to get/set the currently tracked Triumph
* Ability to get/set the currently tracked Pursuit(s)
* Ability to get/set the current tagged Location

**Comment by lowlines** [comment]
Pursuits should already be get/set with the SetItemLockState endpoint. I haven't tested this though, but I know all the state info is there along with the locked/masterwork states etc.

If you could return the `locationHash` and `releaseIndex` that is currently being tracked, that would be REALLY useful for the maps especially in terms of properly mapping things like Public Events where I have to make an educated guess on whether "Public Event Incoming" is actually "Taken Blight" etc.

If you could actually set the tracked map location via the API...that would be huge!

The currently tracked recordHash is being returned with GetProfile[900], but there is currently no way to set that.

**Comment by lowlines** [comment]
An example use case of being able to set the currently tracked locationHash/releaseIndex would be like waiting for the Taken Blight public event to spawn. You can only track it while it is on the in-game map, but it remembers you tracked it even after the public event ends so a person could find it on my maps and "track" it and just do other stuff until the public event shows up without needing to constantly keep checking the director in-game.
The second stage for this would be to actually return a list of "active/visible" locationHashes including public events, but I reckon that is a distant future wishlist type thing.

**Comment by bhollis** [comment]
@lowlines I don't think you can use `SetItemLockState` to track pursuits though. It only does lock/unlock.
